spec/spec.tex

```diff
@@ -712,6 +712,30 @@ \subsubsection{Syntactic Shorthands}
       x <= d
 \end{lstlisting}
 
+To additionally aid readability, a \verb|when| statement whose conditional statement consists of a single line may be combined into a single line.
+If an \verb|else| branch exists, then the \verb|else| keyword must be included on the same line.
+
+The following statement:
+\begin{lstlisting}
+when c :
+   a <= b
+else :
+   e <= f
+\end{lstlisting}
+
+can be equivalently expressed on a single line as follows:
```

Comment:
This is not a single line? Maybe "can have the when branch expressed on a single line" and later "or the whole conditional expressed on a single line?